python/lib/core/dmod/core/common/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+Provides common functions and helper classes
+"""
+
+from .failure import Failure
+from .helper_functions import *
```

Comment:
Would you consider adding `__all__` to `helper_functions` or manually importing each function here to avoid clouting the namespace with imports in `helper_functions`?

Reply:
I'm going to go with the individual imports.

Reply:
Thanks!